### PR TITLE
Searched for rules in index.css only used by getStarted.html, moved t…

### PR DESCRIPTION
**Pull request by baspalinckx**
Searched for rules in index.css only used by getStarted.html
Moved these rules to a new file: getStarted.css
Added an import for this very file in getStarted.html

Issue: https://github.com/shahednasser/sbuttons/issues/485

### CODE REVIEW

File: assets/css/getStarted.css [hunk 1]
```diff
@@ -0,0 +1,54 @@
+.section-header {
```
Comment (shahednasser) [inline assets/css/getStarted.css:1]: this is used by index.html so it needs to be in index.css

File: assets/css/getStarted.css [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  text-align: center;
+}
+
+.secondary-hr {
+  width: 10%;
```
Comment (shahednasser) [inline assets/css/getStarted.css:6]: same for this one

**Review by shahednasser — CHANGES_REQUESTED** [review]

**Comment by baspalinckx** [comment]
Really, I could not find both of these with a CTRL-F search in index.html? But I can imagine that they might be used there in the future. Changed it

**Comment by shahednasser** [comment]
Great work thank you!

@all-contributors please add @baspalinckx for code

**Pull request merged**